schema/Retail/RET/on_search.ts

```diff
@@ -102,6 +102,7 @@ export const onSearchSchema = {
                 },
                 symbol: {
                   type: 'string',
+                  pattern: '^$|^https?:\\/\\/[^\\s]*',
```

Comment:
ajv offers support for the url formats; that format can be used instead of regex